c2corg_ui/templates/index.html

```diff
@@ -74,15 +73,14 @@ <h3 translate>Mobile application</h3>
         <h3 translate>Last forum topics</h3>
         <h4 class="text-danger" translate ng-show="forumCtrl.errorForum">Sorry, there was an error while getting the forum data.</h4>
         <div class="fpost" ng-repeat="post in forumCtrl.topics track by post.id" ng-if="::forumCtrl.topics.length">
-          <a href="${discourse_url}t/{{::post['slug']}}/{{::post['id']}}" title="{{::post['title']}}">
+          <a href="${discourse_url}t/{{::post['slug']}}/{{::post['id']}}" title="{{::post['last_poster_username']}}">
             <div class="fpcontent">
               <div class="fleft">
                 <div class="fportrait">
                   <img ng-src="${discourse_url}{{::post['avatar_template']}}">
                 </div>
                 <div class="ftxt">
-                  <div class="fauthor">{{::post['last_poster_username']}}</div>
-                  <div class="ftitle"><span class="title">{{::post['title']}}</span> - {{::post['last_posted_at'] | amUtc | amLocal | amTimeAgo}}</div>
+                  <div class="ftitle"><span class="title">{{::post['title']}}</span></div>
```

Comment:
Perhaps it depends on the screen resolution/size but I think you have removed to many things in the "mobile app" block and in the "last topics" block => it looks quite empty in my screen
<img width="474" alt="capture d ecran 2017-11-28 a 21 07 29" src="https://user-images.githubusercontent.com/1192331/33341748-32cf061e-d480-11e7-90fe-d2f32a82fe3b.png">

I would keep the "mobile app" introduction. In the "last topics" block I think the "date"/"age" is relevant and consistent with the activity feed cards (that do have a "date"/"age"). In my opinion you have enough space to have the thead title on a first line and the last author + age on a second line (could use a smaller fontsize).

Reply:
Ok I propose to readd the age for screen more than 1600px 

For mobile bloc ... it's a complain of some people, I reask to forum